test/input/beamville/beam.conf

```diff
@@ -136,7 +136,10 @@ beam.calibration.mode.benchmarkFileLoc = ${beam.inputDirectory}"/calibration/ben
 beam.outputs.baseOutputDirectory = ${PWD}"/output/beamville"
 beam.outputs.baseOutputDirectory = ${?BEAM_OUTPUT}
 beam.outputs.addTimestampToOutputDirectory = true
-
+beam.outputs.writeGraphs = true
```

Comment:
Is this even needed? These are the defaults, so there should be no need to add more here.

Reply:
Hi @JustinPihony , Yes it is needed because we have to generate graph based on this property(default value of this property is true). We can discuss further on this with @wrashid or @colinsheppard

Reply:
Yes, but what I was pointing out is that these are the defaults, so don't need set in beam.conf - since they are set in the beamtemplate.

Reply:
Hi @JustinPihony ,Now this property has been removed from beam.conf its exist only in beam-template.conf